src/components/v5/common/ActionSidebar/partials/Motions/steps/RevealStep/RevealStep.tsx

```diff
@@ -83,72 +84,66 @@ const RevealStep: FC<RevealStepProps> = ({
               transform={transform}
               onSuccess={handleSuccess}
             >
-              <div className={clsx({ 'mb-6': !userVoteRevealed })}>
-                <div className="flex items-center justify-between gap-2 mb-2">
-                  <h4 className="text-2">
-                    {formatText({ id: 'motion.revealStep.title' })}
-                  </h4>
-                  <MotionBadge status={isSupportVote ? 'support' : 'oppose'} />
-                </div>
-                <div className="flex items-center justify-between gap-2">
-                  <span className="text-sm text-gray-600">
-                    {formatText({ id: 'motion.revealStep.rewards' })}
-                  </span>
-                  <span className="text-sm">
-                    <Numeral
-                      value={voterReward || '0'}
-                      decimals={decimals}
-                      suffix={symbol}
+              {hasUserVoted ? (
+                <>
+                  <div className={clsx({ 'mb-6': !userVoteRevealed })}>
+                    <div className="flex items-center justify-between gap-2 mb-2">
+                      <h4 className="text-2">
+                        {formatText({ id: 'motion.revealStep.title' })}
+                      </h4>
+                      <MotionBadge
+                        status={isSupportVote ? 'support' : 'oppose'}
+                      />
+                    </div>
+                    <div className="flex items-center justify-between gap-2">
+                      <span className="text-sm text-gray-600">
+                        {formatText({ id: 'motion.revealStep.rewards' })}
+                      </span>
+                      <span className="text-sm">
```

Comment:
You can remove this span and add text class to the wrapper, because both texts inside have same text class